canal/config.go

```diff
@@ -76,6 +77,7 @@ func NewDefaultConfig() *Config {
 	c.ServerID = uint32(rand.Intn(1000)) + 1001
 
 	c.Flavor = "mysql"
+	c.SkipMysqlDump = true
```

Comment:
I don't think set default value as `true` here is a good idea. 

btw: I suggest use the consistent case word `MySQL`.